common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3002,6 +3002,22 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVE_TXN_READONLY_ENABLED("hive.txn.readonly.enabled", false,
       "Enables read-only transaction classification and related optimizations"),
 
```

Comment:
Here I would add a comment like:
Configs having to do with DeltaFilesMetricReporter, which collects lists of most recently active tables with the most number of active/obsolete deltas.